python/dgl/backend/backend.py

```diff
@@ -866,8 +866,8 @@ def zerocopy_from_dgl_ndarray(input):
 # kernels (see kernel.py), and plug into tensor framework using custom op
 # extensions.
 
-def binary_reduce(reducer, binary_op, graph, lhs, rhs, lhs_data, rhs_data,
-                  out_size, lhs_map, rhs_map, out_map):
+def binary_reduce(reducer, op, G, A_target, B_target, A, B,
```

Comment:
I think the argument names are the same as c++ backend, should we refactor c++ impls too?

Reply:
It could be another separate PR I think...  Lots of stuff to change there.